runtime/doc/lua.txt

```diff
@@ -1090,6 +1090,20 @@ Example: >
     vim.g.foo = nil   -- Delete (:unlet) the Vimscript variable.
     vim.b[2].foo = 6  -- Set b:foo for buffer 2
 <
+
+Note setting dictionary fields directly will not write them back into Nvim.
```

Comment:
```suggestion
Note: Setting dictionary fields directly will not write them back into Nvim.
```

Reply:
`Note` without the `:` is also highlighted and is also used throughout the docs, so there isn't a correct choice here.

Reply:
Then I'd add a "Note _that_" ;)